pkg/apis/serving/k8s_validation_test.go

```diff
@@ -1498,7 +1498,7 @@ func TestContainerValidation(t *testing.T) {
 			},
 			want: apis.ErrMultipleOneOf("readinessProbe.exec", "readinessProbe.tcpSocket", "readinessProbe.httpGet"),
 		}, {
-			name: "invalid readiness http probe (has wrong port)",
+			name: "valid readiness http probe with a different container port",
```

Comment:
It would be more user friendly to add the `want: nil` for these two tests, so their result would be explicit. Other tests also do that.

Reply:
Thanks for the feedback, Carlisia. I added it to the tests :)